html/pfappserver/root/static.alt/src/globals/configuration/pfConfigurationMaintenanceTasks.js

```
@@ -0,0 +1,667 @@
+          fields: [
+            pfConfigurationMaintenanceTaskFields.id(context),
+            pfConfigurationMaintenanceTaskFields.description(context),
+            pfConfigurationMaintenanceTaskFields.status(context),
+            pfConfigurationMaintenanceTaskFields.interval(context),
+            pfConfigurationMaintenanceTaskFields.batch(context),
+            pfConfigurationMaintenanceTaskFields.timeout(context)
+          ]
+        }
+      ]
+    case 'switch_cache_lldpLocalPort_description':
+      return [
+        {
+          tab: null, // ignore tabs
+          fields: [
+            pfConfigurationMaintenanceTaskFields.id(context),
+            pfConfigurationMaintenanceTaskFields.description(context),
+            pfConfigurationMaintenanceTaskFields.status(context),
+            pfConfigurationMaintenanceTaskFields.interval(context),
+            pfConfigurationMaintenanceTaskFields.process_switchranges(context)
+          ]
+        }
+      ]
+    case 'certificates_check':
+      return [
+        {
+          tab: null, // ignore tabs
+          fields: [
+            pfConfigurationMaintenanceTaskFields.id(context),
+            pfConfigurationMaintenanceTaskFields.description(context),
+            pfConfigurationMaintenanceTaskFields.status(context),
+            pfConfigurationMaintenanceTaskFields.interval(context),
+            pfConfigurationMaintenanceTaskFields.delay(context),
+            pfConfigurationMaintenanceTaskFields.certificates(context)
+          ]
+        }
+      ]
+    case 'password_of_the_day':
+      return [
+        {
+          tab: null, // ignore tabs
+          fields: [
+            pfConfigurationMaintenanceTaskFields.id(context),
+            pfConfigurationMaintenanceTaskFields.description(context),
+            pfConfigurationMaintenanceTaskFields.status(context),
+            pfConfigurationMaintenanceTaskFields.interval(context)
+          ]
+        }
+      ]
+    default:
+      return [
```

We should probably raise an exception if the task id is not known.